jupyterlab_caip_optimizer/src/index.ts

```diff
@@ -58,7 +59,8 @@ async function activate(app: JupyterFrontEnd) {
   const listWidget = new ListWordsWidget(listWordsService);
   listWidget.addClass('optimizer');
   app.shell.add(listWidget, 'left', { rank: 100 });
```

Comment:
quick question: what does this `rank: 100` do?

Reply:
I am pretty sure it puts the icon button at the top of the side panel.